src/sentry/issues/producer.py

```diff
@@ -58,5 +67,32 @@ def produce_occurrence_to_kafka(
             payload_data["event"] = event_data
         payload = KafkaPayload(None, json.dumps(payload_data).encode("utf-8"), [])
         _occurrence_producer.produce(Topic(settings.KAFKA_INGEST_OCCURRENCES), payload)
+    elif payload_type == PayloadType.STATUS_CHANGE:
+        if not status_change:
+            raise ValueError("status_change must be provided")
+
+        organization = Project.objects.get(id=status_change.project_id).organization
+        if not features.has("organizations:issue-platform-api-crons-sd", organization):
+            return
+
+        grouphash = (
+            GroupHash.objects.filter(
+                project=status_change.project_id,
+                hash=status_change.fingerprint[0],
+            )
+            .select_related("group")
+            .first()
+        )
+        if not grouphash:
+            logger.error(
+                "grouphash.not_found",
+                extra={
+                    "project_id": status_change.project_id,
+                    "fingerprint": status_change.fingerprint,
+                },
+            )
+            return
+
+        status_change.update_status_for_occurrence(grouphash.group)
```

Comment:
Is the idea that we'll send this message to Kafka in a later pr? This api should be accessible via Kafka so that any team that wants to change the status of an issue can do so, regardless of whether they're in the Sentry codebase.

Reply:
Added that change here!